compiler-rt/lib/scudo/standalone/common.h

```diff
@@ -133,18 +133,39 @@ inline void computePercentage(uptr Numerator, uptr Denominator, uptr *Integral,
 
 // Platform specific functions.
 
+#if defined(SCUDO_PAGE_SIZE)
+
+inline constexpr uptr getPageSizeCached() { return SCUDO_PAGE_SIZE; }
+
+inline constexpr uptr getPageSizeSlow() { return getPageSizeCached(); }
+
+inline constexpr uptr getPageSizeLogCached() {
+  return static_cast<uptr>(__builtin_ctzl(SCUDO_PAGE_SIZE));
```

Comment:
How about using `getLog2()`?

Reply:
Unfortunately, I can't do that and keep the function constexpr. I want to keep it constexpr so I can add a static_assert in the Android config code to make sure the hard-coded value is being used.

I did change the common.cpp to use getLog2 function.

Reply:
I see. I was thinking of marking `getLog2` (and `getLeastSignificantSetBitIndex`) as constexpr but that may forbid us adding some DCHECKs there.